db/db_wal_test.cc

```diff
@@ -756,7 +756,8 @@ class RecoveryTestHelper {
   // Manuall corrupt the specified WAL
   static void CorruptWAL(DBWALTest* test, const Options& options,
                          const double off, const double len,
-                         const int wal_file_id, const bool trunc = false) {
+                         const int wal_file_id, MockEnv* mem_env,
+                         const bool trunc = false) {
```

Comment:
db_wal_test was also one of the slowest. sandcastle tests already testing these with postix_env. If you think that is not good enough and travis should also cover that, we can cancel this patch since otherwise the gains of moving fast tests to MEM_ENV are not going to be much.